code/mission/missionbriefcommon.cpp

```diff
@@ -912,7 +934,7 @@ void brief_render_fade_outs(float frametime)
 			if ( fi->fade_anim.first_frame >= 0 ) {
 				fi->fade_anim.sx = bx;
 				fi->fade_anim.sy = by;
-				hud_anim_render(&fi->fade_anim, frametime, 1, 0, 0, 0, GR_RESIZE_MENU);
+				hud_anim_render(&fi->fade_anim, frametime, 1, 0, 0, 0, bscreen.resize, fi->mirror, scale_factor);
```

Comment:
Is this desired to use bscreen.resize, and thus being explicitly different from this_resize in the RESIZE_MENU case?

Reply:
bscreen.resize is an explicit choice, yes. SCPUI can change it when it calls to rendering the briefing map. I don't remember the details but this is the same setup that rendering the regular icons uses. This PR is really just meant to match the fade icon render to the regular.